tests/test_colab.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import collections
+from unittest.mock import patch, MagicMock
+import sys
+
+import anywidget
+
+import IPython.display
+
+
+def fake_module(**kwargs: dict):
+    return collections.namedtuple("module", *kwargs.keys())(**kwargs)
+
+
+@pytest.fixture
+def mock_colab():
+    enable_custom_widget_manager = MagicMock()
+    mock_module = fake_module(enable_custom_widget_manager=enable_custom_widget_manager)
+    with patch.dict(sys.modules, {"google.colab.output": mock_module}):
+        yield mock_module
```

Comment:
very minor suggestion, these two could be replaced with:

```py
@pytest.fixture
def mock_colab(monkeypatch: pytest.MonkeyPatch):
    mock_module = MagicMock()
    monkeypatch.setitem(sys.modules, "google.colab.output", mock_module)
    return mock_module
```

Reply:
nice